packages/http-client-python/generator/test/dev_requirements.txt

```diff
@@ -0,0 +1,5 @@
+-r ../../eng/scripts/ci/dev_requirements.txt
+aiohttp
+pyright==1.1.391
+pytest-asyncio==0.14.0
+requests==2.32.2
```

Comment:
These 3 deps seems already be declared in `../../eng/scripts/ci/dev_requirements.txt`